### PR TITLE
Added User-Agent

### PR DESCRIPTION
**Pull request by bonniecpk**
To indicate Google Ads Doctor application in the Google Ads API request

### CODE REVIEW

File: oauthdoctor/oauth/helper.go [hunk 1]
```diff
@@ -264,13 +266,14 @@ func (c *Config) oauth2Client(code string) (*http.Client, string) {
...
 // getAccount makes a HTTP request to Google Ads API customer account
-// endpoint and parse the JSON response.
+// endpoint and parses the JSON response.
 func (c *Config) getAccount(client *http.Client) (*bytes.Buffer, error) {
 	req, err := http.NewRequest("GET", apiURL+c.CustomerID, nil)
 	if err != nil {
 		return nil, err
 	}
 
+	req.Header.Set("User-Agent", userAgent())
```
Comment (bobhancock) [inline oauthdoctor/oauth/helper.go]: Put header filed names in lowercase, the HTTP2 standard. Some time next year we will be enforcing this.

File: oauthdoctor/oauth/helper.go [hunk 2]
```diff
@@ -307,6 +310,16 @@ func (c *Config) getAccount(client *http.Client) (*bytes.Buffer, error) {
 	return buf, nil
 }
 
+func userAgent() string {
```
Comment (bobhancock) [inline oauthdoctor/oauth/helper.go]: Is there a test for this?

File: oauthdoctor/oauth/helper.go (hunk 1 above)
Comment (bonniecpk) [reply oauthdoctor/oauth/helper.go]: Done

File: oauthdoctor/oauth/helper.go (hunk 2 above)
Comment (bonniecpk) [reply oauthdoctor/oauth/helper.go]: Good catch! Got too excited. =D

Comment (bobhancock) [inline oauthdoctor/oauth/helper.go]: Add function documentation.

**Pull request merged**